docs/src/handle.rst

```diff
@@ -156,6 +156,10 @@ API
     `close_cb` can be `NULL` in cases where no cleanup or deallocation is
     necessary.
 
+    Please ensure that you invoke this function following the execution of
```

Comment:
I don't think this belongs here since it's documented in the page specific to child processes.

Reply:
Okay, I removed it.

Could you explain me why it should not be here specifically, so that I improve my future PRs please ?